### PR TITLE
Cleanup of files, added sections in README and LICENSE

### PR DESCRIPTION
**Pull request by nazguul512**
Cleanup of old dev files  
Cleanup of old debug prints  
Added Disclaimer and License section in README.md file
Added LICENSE file

### CODE REVIEW

File: config.ini
Comment (Silviumuraru90) [inline config.ini:1]: Seems to be part of .gitignore, but still present here, probably because it was already tracked by .git before the .gitignore entry.

Comment (Silviumuraru90) [reply config.ini:1]: Ah, that was the old .gitignore file, nvm :)

File: README.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
 
 At this moment, the bot doesn't trade by itself, it only takes the tickers available from the XTB broker, applies a RSI + BB strategy and sends all tickers for Buy and Sell through a channel in Telegram.
 The original link for the wrapper is here https://github.com/caiomborges/Python-XTB-API but I decided to included it in my files for a quicker setup.
 I'm not gonna lie, my journey in Python is in its infancy steps, so at the moment ChatGPT is a great help to unblock me and improve my code where needed.
 
-
 ## Setup
 
 After installing all the libraries needed (sorry, at this point I don't have enough experience with pip or such, but I will work on making a requirements.txt), you will need to setup a telegram bot channel in order for you to receive the notifications send by the bot.
  - In the API.py wrapper you will need to modify the file at line 553 for which kind of XTB account you are using, demo or real.
```
Comment (Silviumuraru90) [inline README.md:14]: That line might change constantly, not sure if is that good to add a static line nr. here.

**Review by Silviumuraru90 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nazguul512) [reply README.md:14]: in time, i will make it dynamic, but for now, it's just for quick guidance

**Pull request merged**